config.yml

```diff
@@ -62,39 +62,59 @@ LANGUAGE_CODE: de-de
 LMS_HOST: local.overhang.io
 MFE_ACCOUNT_MFE_APP:
   name: account
-  port: 1997
+  port: 3001
   repository: https://github.com/Abstract-Tech/community-theme-account
-  version: community-theme-master
+  version: consistent-deps
```

Comment:
We should standardize a common branch across all the microfrontends.
I was using the `community-theme` branch, which was created from the `open-release/olive.master` branch on every MFE repository.  I think we should probably merge the `consistent-deps` branch into the `community-theme` branch on every MFE and use the `community-theme` branch here, but i'll leave the decision to you @snglth on how to go on with this.